### PR TITLE
add blocklet support

### PR DESCRIPTION
**Pull request by polunzh**

### CODE REVIEW

File: src/cli/blocklet/use/use.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+  const blockletDir = await loadBlocklet({
+    name: blockletName,
+    localBlockletDir: localBlocklet,
+    blocklets,
+  });
+
+  verifyBlocklet(blockletDir);
+
+  const blockletConfig = JSON.parse(
```
Comment (wangshijun) [inline src/cli/blocklet/use/use.js]: possible exception here

File: src/cli/blocklet/use/use.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+const verifyBlocklet = blockletDir => {
+  const blockletJSONPath = path.join(blockletDir, BLOCKLET_CONFIG_FILEPATH);
+
+  if (!fs.existsSync(blockletJSONPath)) {
+    throw new Error(`${BLOCKLET_CONFIG_FILEPATH} file not found`);
+  }
+
+  const blockletConfig = JSON.parse(fs.readFileSync(blockletJSONPath));
+
+  if (!BLOCKLET_GROUPS.includes(blockletConfig.group)) {
```
Comment (wangshijun) [inline src/cli/blocklet/use/use.js]: we should not throw error here, we need a fallback handler for unrecognized blocklets, though the case is rare in reality.

Besides, do we need to verify blocklet scripts and hooks?

File: src/cli/blocklet/use/use.js [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+/**
+ * Load handler by blocklet group name
+ * @param {string} group - blocklet group name
+ * @return {BaseHandler} handler
+ */
+const getHandlerByBlockletGroup = group => {
+  const filePath = path.join(__dirname, '..', 'lib', 'handlers', `${group}.js`);
+  if (!group || !fs.existsSync(filePath)) {
+    throw new Error(`group ${group} is invalid`);
```
Comment (wangshijun) [inline src/cli/blocklet/use/use.js]: same here: have a fallback here

**Review by wangshijun — APPROVED** [review]

File: src/cli/blocklet/use/use.js (hunk 1 above)
Comment (polunzh) [reply src/cli/blocklet/use/use.js]: added `try...catch`

File: src/cli/blocklet/use/use.js (hunk 2 above)
Comment (polunzh) [reply src/cli/blocklet/use/use.js]: Removed, it seems a bit redundant.

**Pull request merged**